OpenEdXMobile/src/main/java/org/edx/mobile/util/Config.java

```diff
@@ -406,6 +406,25 @@ public boolean isEnabled() {
         }
     }
 
+    public static class FirebaseConfig {
+        @SerializedName("ENABLED")
+        private boolean mEnabled;
+
+        @SerializedName("ANALYTICS_ENABLED")
+        private boolean mAnalytics;
+
+        @SerializedName("CLOUD_MESSAGING_ENABLED")
+        private boolean mCloudMessaging;
+
+        public boolean isEnabled(){
```

Comment:
this function name should be updated to `isAnalyticsEnabled()`

Reply:
That would not be a correct name, because this is the enable for Firebase not just the analytics.